src/directives/fixed-header-table/fixed-header-table.directive.ts

```diff
@@ -105,6 +109,9 @@ export class FixedHeaderTableDirective<T> implements OnInit, OnDestroy {
         if (delta < 1) {
             this.tablePaging.emit();
         }
+
+       // update the class based on the scroll position
+        scrollTop > 0 ? this._hasScrolled = true : this._hasScrolled = false;
```

Comment:
This seems like a reasonable approach to me, but I would reverse this statement:
```ts
this._hasScrolled = (scrollTop > 0);
```